pkg/cmd/server/start/start_master.go

```diff
@@ -558,6 +558,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunEndpointController()
 		kc.RunNamespaceController()
 		kc.RunPersistentVolumeClaimBinder()
+		kc.RunPersistentVolumeProvisioner(pvKClient)
```

Comment:
You're right.  I wasn't thinking about updating PVs but more what kind of pods it requires (none at present).

The binder's permissions are similar to the provisioner's.